lib/options/sort-order.js

```diff
@@ -283,7 +283,44 @@ module.exports = {
             propertyName = null;
             // Look for includes:
             if (node.get(i).is('include')) {
-                propertyName = '$include';
+                // Divide `include` into `$extend`,
+                // mixins with specific name (e. g. `$include breakpoint`),
+                // and the rest — `$include`.
+                var includeType;
+                var mixinName;
```

Comment:
I'm a bit confused with this var's name, since there is actually `@mixin` in Sass but here we're dealing with `@include`